.github/workflows/auto-merge-prs.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
What does this change?

Reply:
So the node 16 actions have been deprecated by Github Actions and the new versions of these actions have transitioned to node 20 (https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/)